yt/yt/server/lib/job_proxy/config.h

```diff
@@ -168,6 +168,7 @@ class TJobProxyInternalConfig
     std::vector<TBindConfigPtr> Binds;
 
     std::vector<TString> GpuDevices;
+    std::vector<int> GpuIndexes;
```

Comment:
What difference between GpuDevices and GpuIndexes? We cannot reuse GpuDevices here?

Reply:
GpuDevices are device paths like /dev/nvidia0 and are used for device bindings in Porto. Nvidia expects device indices or guids to be passed into NVIDIA_VISIBLE_DEVICES.

Reply:
I guess each node report set of available GPUs and expects them in job-spec?

It seems these ids works as opaque identifiers handled in job-env specific ways.
Let's keep one set of these identifiers.

Reply:
Node reports the number of GPUs available and receives number of GPU to allocate in job spec. After that, GPU manager decides which devices should be used for a particular job and passes device identifiers further to job environment.

For Porto job environment, GPU device paths are passed to Job Proxy and Job Proxy binds these devices into User Job container when starts it.
For CRI job environment, combined container is started with NVIDIA_VISIBLE_DEVICES env variable containing indices of the devices used.